api/admincontract.go

```diff
@@ -52,5 +55,11 @@ func NewAdminContractService(runner *Runner) *AdminContractService {
 }
 
 func (cs *AdminContractService) Call(req *http.Request, args *requester.Params, requestBody *rpc.RequestBody, result *requester.ContractResult) error {
-	return wrapCall(cs.runner, cs.allowedMethods, req, args, requestBody, result)
+	ctx, instr := instrumenter.NewMethodInstrument("AdminContractService.call")
+	defer instr.End()
+
+	logger := inslogger.FromContext(ctx)
+	logger.Infof("[ AdminContractService.call ] Incoming request: %s", req.RequestURI)
```

Comment:
logger here w/o traceID

Reply:
`TraceID` is created and assigned to `ctx` in `NewMethodInstrument`